### ISSUE TITLE
JSON undefined at login window

### ISSUE THREAD

**Issue by christianhanlon** [SourceAudio/sa-adobe-panel-beta#46]
Premier 2018 12.1.2/ High Sierra 10.13.16

After installing, cannot log in to main window.  Once panel opens, "Run Script Error" dialog shows up with JSON is undefined.  Does not allow user to input their user information (though they can see the log in window).

**Comment by UltimateBrent** [comment]
After this happens, open up a browser and go to: `http://localhost:7778` click SA Panel, and let me know if there are any errors in the console, please!

**Comment by christianhanlon** [comment]
Hi Brent. 

Unfortunately, this is happening at the client level and they’re a bit confused about opening the local host inspect window. We have also noticed that apart from people who work at our company (Temp Love) our client users are not able to log on to Adobe Panel.  We tried the logins ourselves and if the user email doesn’t originate from @templove.com, the sign page refreshes after entry and doesn’t connect the user. 

Any thoughts?
